### ISSUE TITLE
Release new version?

### ISSUE THREAD

**Issue by devmotion** [PieterjanRobbe/GaussianRandomFields.jl#9]
Is it possible to release a new version? I just noticed that I cannot install GaussianRandomFields on Julia 1.0.1, maybe this package is affected by https://discourse.julialang.org/t/package-compatibility-caps/15301?

**Comment by PieterjanRobbe** [comment]
See [https://github.com/JuliaLang/METADATA.jl/pull/18806](https://github.com/JuliaLang/METADATA.jl/pull/18806). Travis gives
```
ERROR: LoadError: New tag 1.0.1 of package GaussianRandomFields requires julia 0.7.0, but version 1.0.0 of GaussianRandomFields requires julia 0.6.0. Use a new minor package version when support for an old version of Julia is dropped.
```

Any ideas on how to fix this?

**Comment by devmotion** [comment]
I've never seen this error message before but I guess you can fix it by tagging a 1.1.0 or 2.0.0 release. According to [PkgDev](https://github.com/JuliaLang/PkgDev.jl):
> If you drop support for an older version of Julia, you should make at least a minor version bump even if there were no API changes.
